test/hotspot/gtest/runtime/test_os_windows.cpp

```diff
@@ -703,6 +703,22 @@ TEST_VM(os_windows, reserve_memory_special) {
   TestReserveMemorySpecial_test();
 }
 
+TEST_VM(os_windows, processor_count) {
```

Comment:
This test requires the new flag to be set to True, right? You need to configure the test to only run if the flag is True. I'm not sure how to do that with gtests.

Reply:
I've added a jtreg test to run with the product flag enabled.